modules/custom_operations/user_ie_extensions/CMakeLists.txt

```diff
@@ -2,6 +2,10 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
+if("${CMAKE_SOURCE_DIR}" STREQUAL "${CMAKE_CURRENT_SOURCE_DIR}")
```

Comment:
Needs to update the instructions for separate module build: https://github.com/openvinotoolkit/openvino_contrib/tree/master/modules/custom_operations#build-custom-openvino-operation-extension-library